tools/autograd/gen_variable_type.py

```diff
@@ -44,6 +44,18 @@
     'conv_transpose2d', 'conv_transpose3d',
 }
 
+# These functions have their names recorded under trace renamed,
+RENAME_TRACE = {
```

Comment:
is there a plan to get rid of these?

Reply:
Yeah, the plan on record is https://github.com/pytorch/pytorch/issues/8495